### PR TITLE
Fixed OpenCV2 includes and namespaces

### PR DESCRIPTION
**Pull request by doumdi**
Hello Ronan, here is a small patch to make ros_opentld work.

### CODE REVIEW

File: src/libopentld/tld/Clustering.h [hunk 1]
```diff
@@ -35,6 +35,8 @@
 namespace tld
 {
 
+using namespace std;
```
Comment (Ronan0912) [inline src/libopentld/tld/Clustering.h:38]: Can it be moved to the cpp file?

File: src/libopentld/tld/ForegroundDetector.h [hunk 2]
```diff
@@ -36,6 +36,8 @@
 namespace tld
 {
 
+using namespace std;
```
Comment (Ronan0912) [inline src/libopentld/tld/ForegroundDetector.h:39]: same

File: src/libopentld/tld/Clustering.h (hunk 1 above)
Comment (doumdi) [reply src/libopentld/tld/Clustering.h:38]: Yes!

File: src/libopentld/tld/ForegroundDetector.h (hunk 2 above)
Comment (doumdi) [reply src/libopentld/tld/ForegroundDetector.h:39]: Yes!

**Comment by Ronan0912** [comment]
Thanks for the patch!

**Pull request merged**